support-docs/examples.md

```diff
@@ -1935,3 +1797,223 @@ PMIC Example - Limited pins
         }
     ]
 }
+```
+
+Sensor Example - Limited pins 
+
+```json     
+{
+    "componentID": {
+        "partType": "accelerometer",
+        "manufacturer": "ACME Components",
+        "componentName": "abc100",
+        "orderableMPN": "abc100,456",
+        "sourceDatasheetID": {
+            "publishedDate": "2022-03-23",
+            "datasheetURI": "www.acmecomponents.com/abc100datasheet"
+        },
+        "digitalDatasheetID": {
+            "publishedDate": "2023-03-23",
+            "guid": "3e4cd9de-657a-41ae-902e-beca95aff51d"
+        },
+        "status": "active"
+    },
+    "accelerationRanges": 8,
+    "accelerationSensitivity": {
+        "value": {
+            "typValue": 256,
+            "maxvalue": 257
+        },
+        "conditions": "max acceleration range is 8g"
+    },
+    "accelerationSensitivityOverTemperature": {
```

Comment:
I'm also thinking maybe it would be worth listing the units as part of "definitions.json#/value". I know it's part of the spec, but I think it makes it harder to read the datasheet here because I don't know what units these things are in.

Reply:
As discussed, let's revisit after the "units" discussion in the next meeting.